marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -643,7 +643,7 @@ public void exportProject(OutputStream outputStream) {
 
     private void writeToStream(OutputStream out) {
         try (ZipOutputStream zout = new ZipOutputStream(out)){
-            Stream.of("entities", "flows", "src" + File.separator + "main", "mappings", "step-definitions", "gradle",
+            Stream.of("entities", "flows", "src" + File.separator + "main", "mappings", "step-definitions", "loadData", "gradle",
```

Comment:
@aebadirad @ryanjdew Do you know the history of why these folders aren't under "src/main" to begin with? If they were, we'd only need to grab "src/main" here.

Reply:
Also, the name "loadData" hurts my head, since it's a "verb + noun" and everything else is "nouns". Guess that ship has sailed though.

Reply:
artifacts != code, but that's why I designed it that way. Current trajectory seems to be more 'java' oriented, so it may make sense to change that in the future.